match/lib/match/storage/google_cloud_storage.rb

```diff
@@ -152,9 +152,10 @@ def ensure_keys_file_exists(google_cloud_keys_file)
           return google_cloud_keys_file
         end
 
-        if File.exist?(DEFAULT_KEYS_FILE_NAME)
-          return DEFAULT_KEYS_FILE_NAME
-        end
+        return DEFAULT_KEYS_FILE_NAME if File.exist?(DEFAULT_KEYS_FILE_NAME)
+
+        fastlane_subfolder = File.join(FastlaneCore::FastlaneFolder.path, DEFAULT_KEYS_FILE_NAME)
```

Comment:
Should this be called `fastlane_subfolder` if it's a file path? How about `fastlane_folder_gc_keys_path` or `gc_keys_path`? It's a bit more verbose, but I think it makes more sense semantically.

Reply:
Sounds good